### ISSUE TITLE
You and you have been sent a personal message

### ISSUE THREAD

**Issue by jdarwood007** [SimpleMachines/SMF#7679]
#### Description
This is mostly a grammatical issue.

```
$txt['new_pm_tolist_body'] = 'You and {TOLIST} have just been sent a personal message by {SENDER} on {FORUMNAME}.
```
```
$txt['new_pm_body_tolist_body'] = 'You and {TOLIST} have just been sent a personal message by {SENDER} on {FORUMNAME}.
```

SMF does not try to figure out and remove your name from the {TOLIST}, resulting in the notification saying you and {Your display Name} in the notification.

Should we even fix this?

### Steps to reproduce
1. Receive a PM that was sent to a group of people and you have notifications turned on

**Comment by Sesquipedalian** [comment]
The best approach would probably be to remove the hard coded "You and" from the string and just use the complied recipients list as provided.

**Comment by jdarwood007** [comment]
Agreed.  Maybe in the future we could.  But it would require sending each notification individually as we would have to parse out which person was you in that list.
